includes/class-wc-skroutz-analytics-tracking.php

```diff
@@ -116,6 +118,12 @@ private function prepare_order_data() {
 			'tax'      => $this->calculate_order_tax(),
 		);
 
+		$payment_gateway = wc_get_payment_gateway_by_order( $this->order );
+		if( $payment_gateway ) {
```

Comment:
https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#space-usage
```suggestion
		if ($payment_gateway) {
```